src/components/ui/form/TextInput/style.scss

```diff
@@ -0,0 +1,10 @@
+input {
+    background-color: rgb(249 250 251);
+    border: 1px solid rgb(209 213 219);
```

Comment:
Noticed that you have used rem. So can we be consistent here also with same format?

Reply:
I've used `px` here just because I don't want it to be changed based on the root. I want the `border-width` to be fixed. Please suggest me the best practice here.